internal/service/redshift/cluster.go

```diff
@@ -60,6 +60,10 @@ func ResourceCluster() *schema.Resource {
 				Optional:     true,
 				Computed:     true,
 				ValidateFunc: validation.StringInSlice(redshift.AquaConfigurationStatus_Values(), false),
+				Deprecated:   "This parameter is no longer supported by the AWS API. It will be removed in the next major version of the provider.",
```

Comment:
Do we need to add a CHANGELOG entry for this?

Reply:
Yes, we should

Reply:
And the bug fix, too